internal/cmd/cmd.go

```diff
@@ -19,11 +19,22 @@ import (
 	"github.com/GoogleCloudPlatform/serverless-sample-tester/internal/sample"
 	"github.com/GoogleCloudPlatform/serverless-sample-tester/internal/util"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 	"log"
 	"os/exec"
 	"path/filepath"
 )
 
+// InitConfig initializes binds command line flags and environment variables to viper configuration keys. This should
```

Comment:
This is also a placeholder for @danielylee01 's work, to be built off of this PR: https://github.com/GoogleCloudPlatform/serverless-sample-tester/pull/22